docs/reproduce/neural_chat_v3-3_workflow.md

```diff
@@ -0,0 +1,97 @@
+# Step-by-Step
+
+This is a step-by-step tutorial to obtain the extreme inference speed and accuracy of [Intel/neural-chat-7b-v3-3](https://huggingface.co/Intel/neural-chat-7b-v3-3), we recommend using [AutoRound](https://github.com/intel/auto-round.git) for quantization and [Neural Speed](https://github.com/intel/neural-speed.git) for inference. 
+
+
+# Prerequisite​
+We recommend use the latest version of [Intel-Extension-for-Transformers v1.3.1](https://pypi.org/project/intel-extension-for-transformers/1.3.1/) and [NeualSpeed v0.2.dev0](https://pypi.org/project/neural-speed/0.2.dev0/)
+
+```bash
+# install neural-speed
+pip install neural-speed==0.2.dev0
+# install intel-extension-for-transformers
+pip install intel-extension-for-transformers==1.3.1
```

Comment:
autoround?

Reply:
installed when quantization:

cd examples/huggingface/pytorch/text-generation/quantization/auto_round
pip install -r requirements.txt
bash run_autoround.sh

Reply:
https://github.com/intel/intel-extension-for-transformers/blob/main/examples/huggingface/pytorch/text-generation/quantization/auto_round/requirements.txt#L12